modules/relation/src/main/RelationActor.scala

```diff
@@ -20,6 +21,11 @@ private[relation] final class RelationActor(
 
   private var onlines = Map[ID, LightUser]()
 
+  override def preStart(): Unit = {
+    context.system.lilaBus.subscribe(self, 'users, 'startGame)
+    context.system.lilaBus.subscribe(self, 'users, 'finishGame)
+  }
+
```

Comment:
don't forget to unsubscribe on postStop! https://github.com/ornicar/lila/blob/master/modules/simul/src/main/Socket.scala#L29-L32